testing/citest/spinnaker_testing/spinnaker_test_scenario.py

```diff
@@ -423,18 +466,19 @@ def __init__(self, bindings, agent=None):
       agent: [SpinnakerAgent] The Spinnaker agent to bind to the scenario.
     """
     super(SpinnakerTestScenario, self).__init__(bindings, agent)
-    agent = self.agent
+    agent=self.agent
```

Comment:
I fixed that in the commit we are preparing.  Just finishing migrating to your new bindings and init and hope to have it in next day or two.